### ISSUE TITLE
Unable to handle management route with forward slash

### ISSUE THREAD

**Issue by joebride** [F5Networks/f5-declarative-onboarding#267]
### Environment
 * Declarative Onboarding Version: v1.23.0, v1.24.0
 * BIG-IP Version: 15.1.3.0.148.11

### Summary
I would like to setup SNMP (SnmpUser, SnmpAgent, SnmpTrapDestination) and several ManagementRoutes via DO, but it fails with 404 error-code.
Please note, that Management-IP and default management-route is preconfigured by DHCP.
In addition more specific management-routes (static ones) are also exist.

The example shows DO with SNMPUser which is also not working, getting 404 error code.

### Steps To Reproduce
1. Submit the following declaration:
```json
            "Common": {
                "IAS_RO_ACCESS_1": {
                    "access": "ro",
                    "authentication": {
                        "protocol": "sha"
                    },
                    "class": "SnmpUser",
                    "oid": ".1.3",
                    "privacy": {
                        "protocol": "aes"
                    }
                },
                "class": "Tenant"
            },
            "async": true,
            "class": "Device",
            "label": "Configure SNMP-user,-allowlist and v3-traps with two trap-receivers. Set management-routes, preserve DHCP-routes",
            "schemaVersion": "1.23.0"
        },

```

2. Observe the following error response:
```json
       "id": "851a137b-5b94-4303-855d-d980ae9f28aa",
        "message": "invalid config - rolled back",
        "result": {
            "class": "Result",
            "code": 404,
            "errors": [
                "tryUntil: max tries reached: 01020036:3: The requested Management Route (/Common/32) was not found.",
                "tryUntil: max tries reached: 01020036:3: The requested Management Route (/Common/32) was not found."
            ],
            "message": "invalid config - rolled back",
            "status": "ERROR"
        },
        "selfLink": "https://localhost/mgmt/shared/declarative-onboarding/task/851a137b-5b94-4303-855d-d980ae9f28aa",
        "status": "ERROR"
    }
}


```

### Expected Behavior
DO should be able to succeed.

### Actual Behavior
Unable to setup SNMP or static management-routes via DO.
When I add to DO-declaration the system-parameter "preserveOrigDhcpRoutes": true' the error-message is different:
01070734:3: Configuration error: invalid Management Route, the dest/netmask pair 100.125.4.0/255.255.255.0 already exists for /Common/100.125.4.0_24",

**Comment by dstokesf5** [comment]
Can you gather and provide trace files to help us better identify the issue? https://clouddocs.f5.com/products/extensions/f5-declarative-onboarding/latest/declarations/miscellaneous.html#enabling-traces-in-do-responses

**Comment by joebride** [comment]
I created traces, but need a secure channel to share it.

It seems that DO cannot correctly manage management-routes with Slash in its name.
See example:

tmsh-configuration:
```json 
sys management-route 100.125.4.0/24 {
    description XYZ
    gateway 10.91.42.1
    network 100.125.4.0/24
}
```
will be wrongly "translated" to:
```json 
    "ManagementRoute": {
      "24": {
        "name": "24",
        "mtu": 0,
        "network": "100.125.4.0/24",
        "gateway": "10.91.42.1"
      },
      "32": {
        "name": "32",
        "mtu": 0,
        "network": "10.14.252.252/32",
        "gateway": "10.91.42.1"
      },
      "default": {
        "name": "default",
        "mtu": 0,
        "network": "default",
        "gateway": "10.91.42.1"
      }
    },
```

**Comment by dstokesf5** [comment]
That might be enough information for me to move forward with this issue. If it is not, I will reach out to you to determine a secure way to share the trace files.

**Comment by dstokesf5** [comment]
That was enough to reproduce the failure. Thank you for your feedback. I have added this issue to our internal product backlog as AUTOTOOL-2773.

**Comment by dstokesf5** [comment]
This has been resolved in the 1.26.0 release.
